packages/react/src/components/Animation/Animation.tsx

```diff
@@ -77,6 +75,39 @@ export interface AnimationProps
    * - cubic-bezier(n,n,n,n) - Lets you define your own values in a cubic-bezier function
    */
   timingFunction?: string
+
+  /** Show the component; triggers the enter or exit animation. */
+  visible?: boolean
+
+  /** Run the enter animation when the component mounts, if it is initially shown. */
+  appear?: boolean
+
+  /** Wait until the first "enter" transition to mount the component (add it to the DOM). */
+  mountOnEnter?: boolean
+
+  /** Unmount the component (remove it from the DOM) when it is not shown. */
+  unmountOnExit?: boolean
+
+  /** The duration of the transition, in milliseconds. */
+  timeout?: number | { enter?: number; exit?: number; appear?: number }
```

Comment:
based on your comment, should `timeout` actually be named `duration`?

Reply:
ah, nevermind. i see that it is what it's called in `react-transition-group`... https://reactcommunity.org/react-transition-group/transition#Transition-prop-timeout

Reply:
Yeah, this two are different concepts (duration is the css animationDuraion), and this is timeout for removing the elements. Maybe I should come up with a better name...